spring-integration-zeromq/src/main/java/org/springframework/integration/zeromq/dsl/ZeroMqMessageProducerSpec.java

```diff
@@ -108,6 +109,19 @@ public ZeroMqMessageProducerSpec topics(String... topics) {
 		return this;
 	}
 
+	/**
+	 * Specify if the topic
+	 * that {@link SocketType#SUB} socket is going to receive is wrapped with an additional empty frame.
+	 * It is ignored for all other {@link SocketType}s supported.
+	 * @param wrapTopic true iff the received topic is wrapped with an additional empty frame.
```

Comment:
DITTO.
Plus I believe on the consumer side this property better to call `unwrap`.